src/components/card/card.tsx

```diff
@@ -0,0 +1,192 @@
+     */
+    @Prop()
+    public actions?: Array<ActionBarItem | ListSeparator> = [];
+
+    /**
+     * When true, improve the accessibility of the component and hints the user
+     * that the card can be interacted width.
+     */
+    @Prop({ reflect: true })
+    public clickable: boolean = false;
+
+    /**
+     * The orientation of the card,
+     * specially useful when the card has an image.
+     */
+    @Prop({ reflect: true })
+    public orientation: 'landscape' | 'portrait' = 'portrait';
+
+    /**
+     * Fired when a action bar item has been clicked.
+     */
+    @Event()
+    public actionSelected: EventEmitter<ActionBarItem>;
+
+    public render() {
+        return (
+            <section tabindex={this.clickable ? 0 : ''}>
+                {this.renderImage()}
+                <div class="body">
+                    {this.renderHeader()}
+                    {this.renderSlot()}
+                    {this.renderValue()}
+                    {this.renderActionBar()}
+                </div>
+            </section>
+        );
+    }
+
+    private renderImage() {
+        if (!this.image) {
+            return;
+        }
+
+        return <img src={this.image.src} alt={this.image.alt} loading="lazy" />;
+    }
+
+    private renderHeader() {
+        if (!this.heading && !this.subheading && !this.icon) {
+            return;
+        }
+
+        return (
+            <header>
+                {this.renderIcon()}
+                <div class="headings">
+                    {this.renderHeading()}
+                    {this.renderSubheading()}
+                </div>
+            </header>
+        );
+    }
+
+    private renderIcon() {
+        const icon = getIconName(this.icon);
+        const color =
+            typeof this.icon !== 'string' ? this.icon?.color : undefined;
+
+        if (!icon) {
+            return;
+        }
+
+        return (
+            <limel-icon
+                style={{
+                    color: `${color}`,
+                }}
+                badge={true}
+                name={icon}
+            />
+        );
+    }
+
+    private renderHeading() {
+        if (!this.heading) {
+            return;
+        }
+
+        return <h1>{this.heading}</h1>;
+    }
+
+    private renderSubheading() {
+        if (!this.subheading) {
+            return;
+        }
+
+        return <h2>{this.subheading}</h2>;
+    }
+
+    private renderSlot() {
+        return <slot name="component" />;
+    }
+
+    private renderValue() {
+        return <limel-markdown value={this.value} />;
+    }
+
+    private handleActionSelect = (
+        event: CustomEvent<ActionBarItem | ListSeparator>,
+    ) => {
+        event.stopPropagation();
+        if (isItem(event.detail)) {
+            this.actionSelected.emit(event.detail);
+        }
+    };
+
+    private renderActionBar() {
+        if (!this.actions.length) {
+            return;
+        }
+
+        return (
+            <limel-action-bar
```

Comment:
I must say that I'm not super happy about using `limel-action-bar` for rendering actions, as in most scenarios in card components, there is a "Primary" action, and actions are more highlighted. 
1. We don't have a way to make one of the actions primary in the action bar. 
2. The primary action is always at the right side, however, it might be so that it ends up in the overflow menu, when the card becomes too small. That also, I don't know how to handle yet. 

These are not easy to fix, without overcomplicating the APIs of `limel-action-bar`.

But I think for now, this can work. For future, we should probably make a custom actions sections inside the `limel-card`, instead.